tests/unit/components/gh-notification-test.js

```diff
@@ -7,44 +7,44 @@ import {
 from 'ember-mocha';
 import sinon from 'sinon';
 
-describeComponent(
-    'gh-notification',
-    'Unit: Component: gh-notification',
-    {
-        unit: true
-        // specify the other units that are required for this test
-        // needs: ['component:foo', 'helper:bar']
-    },
-    function () {
-        it('closes notification through notifications service', function () {
-            let component = this.subject();
-            let notifications = {};
-            let notification = {message: 'Test close', type: 'success'};
+// describeComponent(
```

Comment:
Does this test need to be commented out? It should be untouched by the other changes in this PR, interested to hear if it's causing issues